llvm/include/llvm/IR/ConstantRangeList.h

```diff
@@ -0,0 +1,86 @@
+//===- ConstantRangeList.h - A list of constant range -----------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Represent a list of signed ConstantRange and do NOT support wrap around the
+// end of the numeric range. Ranges in the list are ordered and no overlapping.
```

Comment:
```suggestion
// end of the numeric range. Ranges in the list are ordered and not overlapping.
```
or "non-overlapping".

Reply:
Done!